tough/src/schema/mod.rs

```diff
@@ -1128,11 +1128,18 @@ pub struct Timestamp {
 /// file, this MUST only include a description of the snapshot.json file.
 #[derive(Debug, Clone, Deserialize, Serialize, Eq, PartialEq)]
 pub struct TimestampMeta {
-    /// The integer length in bytes of the snapshot.json file.
-    pub length: u64,
+    /// The integer length in bytes of the snapshot.json file. It is OPTIONAL and
+    /// can be omitted to reduce the snapshot metadata file size. In that case the client MUST use a
+    /// custom download limit for the listed metadata.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub length: Option<u64>,
```

Comment:
After this change, `TimestampMeta` looks identical to `SnapshotMeta`. If we're taking an API break for this, seems like we should just converge on a single `MetaFile` struct, to better convey the idea that this is actually the same type according to the spec.

Reply:
I agree, do you want me to attempt to perform this change now or would you prefer a dedicated PR later?

Reply:
I think this should probably be in the same PR. Perhaps a separate commit for now and then we squash at the end of the review process?

Reply:
I've just pushed a commit that replaces `SnapshotMeta` and `TimestampMeta` with `Metafiles`.

I'm open to suggestions for a better name :sweat_smile: